http2/src/test/java/com/blazemeter/jmeter/http2/sampler/HTTP2StreamHandlerTest.java

```diff
@@ -24,183 +25,202 @@
+    http2StreamHandler.onData(stream, dataFrame, callback);
 
-        ByteBuffer b = ByteBuffer.wrap(data.getBytes());
+    resSR = http2StreamHandler.getHTTP2SampleResult();
 
-        Mockito.when(dataFrame.getData())
-                .thenReturn(b);
+    assertTrue(resSR.isSuccessful());
+  }
 
-        Mockito.when(dataFrame.isEndStream())
-                .thenReturn(true);
+  @Test
+  public void onResetTest() {
 
-        http2SampleResult = new HTTP2SampleResult();
+    Mockito.when(resetFrame.getError()).thenReturn(0x7);
 
-        http2StreamHandler = new HTTP2StreamHandler(http2Connection, url, null, null,
-                http2SampleResult);
+    http2SampleResult = new HTTP2SampleResult();
+    http2StreamHandler = new HTTP2StreamHandler(http2Connection, url, null,
+        null, http2SampleResult);
+    HTTP2SampleResult resSR = http2StreamHandler.getHTTP2SampleResult();
```

Comment:
isn't resSR the same instance as http2SampleResult? If so then just use that already defined variable and no need to retrieve a new one.